tests/test_cqcconnection.py

```diff
@@ -62,8 +78,727 @@ def commands_to_apply_simple_h(cqc):
     q.H()
 
 
+def commands_to_apply_bit_flip_code(cqc):
```

Comment:
Could you move these tests to a seperate file in the folder `test_cases_cqcconnection`, see the existing file there, to not have to many test cases in this file. But please leave the one that was already here as an example of how to write these tests.